### ISSUE TITLE
Edit when user tabs on an image?

### ISSUE THREAD

**Issue by liuzhen2008** [wix/react-native-wordpress-editor#8]
After  an image is picked and rendered on the editor.

If the user were to touch the image, the image will blur and show an edit button. However, currently, clicking on that button doesn't do anything.

Is there a navigator event fired for that? Or can we optionally not to have that behaviour? 

Observed on android.

**Comment by janroures** [comment]
have you made it to the point of rendering a picked image? Whenever I pick one, all I get is a blank square after setting the uri. can you tell me please how you did it? thanks!

**Comment by liuzhen2008** [comment]
@janroures  
`EditorManager.addImages([{url: response.uri}]);` is all thats needed. (Assuming the response is from some image picker and it has a file uri). Note that it is an array of object with attribute name "url"

**Comment by janroures** [comment]
That's weird. It's the same exact code I got, and I know the uri is right cause I used it in other projects. Will keep investigating.

**Comment by janroures** [comment]
All I see is this (both simulator and device): 
![img_8790](https://cloud.githubusercontent.com/assets/8616333/18976975/be271a9e-86b6-11e6-89d2-cddf77f017a7.PNG)

**Comment by liuzhen2008** [comment]
what iOS version? @janroures

**Comment by janroures** [comment]
@liuzhen2008 9.3.5

**Comment by liuzhen2008** [comment]
@janroures Did you hard code the uri to show up in the renderer? Did you actually have image permission and stuff setup?

**Comment by janroures** [comment]
no, I'm using the exact same code as you  `EditorManager.addImages([{url: response.uri}]);` when I get the response from the picker. @liuzhen2008

**Comment by janroures** [comment]
Oh. My bad man. I had {uri: response.uri} instead of {url: response.uri}. ^^

**Comment by janroures** [comment]
How did you manage to get the post data after adding an image? After I call `EditorManager.getPostData()` I always get this weird error: `Exception '*** -[__NSCFString rangeOfString:options:range:locale:]: nil argument' was thrown while invoking getPostData on target RNWordPressEditorManager with params (
    18,
    19
)
`

Ever seen that before?

**Comment by liuzhen2008** [comment]
@janroures Have you get around that issue by now?

**Comment by tendolukwago** [comment]
@liuzhen2008 @janroures  have either of you figured this issue out?

**Comment by liuzhen2008** [comment]
@tendolukwago  nope

**Comment by tendolukwago** [comment]
@liuzhen2008 @janroures I ended up commenting out these lines:

```
 NSDictionary *coverImageData = [activeEditor coverImageData];
  if (coverImageData != nil)
  {
    result[@"coverImage"] = coverImageData;
  }
```
in RNWordPressEditorManager.m

I don't know what coverImageData is used for, so unless I run into any issues, I'm rolling with this.
